src/jdk.management/share/native/libmanagement_ext/management_ext.c

```diff
@@ -31,9 +31,9 @@
 
 #define ERR_MSG_SIZE 128
 
-const JmmInterface* jmm_interface = NULL;
+const JmmInterface* jmm_interface_management_ext = NULL;
```

Comment:
Can you add a comment before declaring the two "exported" symbols together:
```
// These symbols are global in this library but need to be uniquely named to avoid conflicts
// with same-named symbols in other libraries, when statically linking.
```
Thanks.

Reply:
Oops! Sorry meant to add this comment to the declarations in the hpp file.

Reply:
Added comment as suggested (with minor adjustment) in management_ext.h. Thanks.